### PR TITLE
Fix the description to define endpoint properly

### PR DESCRIPTION
**Pull request by haruki-yamanashi**
Fix and add follwoing arguments when define endpoint.
- publicurl
- region

### CODE REVIEW

**Comment by haruki-yamanashi** [comment]
@samirjorina @digitalfishpond

**Comment by witekest** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -120,9 +120,9 @@ setup:
 openstack service create --name monasca monitoring
 openstack service create --name logs logs
 openstack service create --name logs_v2 logs_v2
-openstack endpoint create monasca public http://192.168.10.6:8070/v2.0
-openstack endpoint create logs public http://192.168.10.6:5607/v3.0
-openstack endpoint create logs_v2 public http://192.168.10.6:5607/v2.0
+openstack endpoint create monasca --publicurl http://192.168.10.6:8070/v2.0 --region <Region_name>
```
Comment (samirjorina) [inline README.md]: this does not work for me:

```
$ openstack endpoint create monasca --publicurl http://192.168.10.6:8070/v2.0 --region RegionOne
usage: openstack endpoint create [-h] [-f {json,shell,table,value,yaml}]
                                 [-c COLUMN] [--max-width <integer>]
                                 [--fit-width] [--print-empty] [--noindent]
                                 [--prefix PREFIX] [--region <region-id>]
                                 [--enable | --disable]
                                 <service> <interface> <url>
openstack endpoint create: error: argument <interface>: invalid choice: u'http://192.168.10.6:8070/v2.0' (choose from 'admin', 'public', 'internal')
$ openstack --version
openstack 3.12.0
```

**Review by samirjorina — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (haruki-yamanashi) [reply README.md]: Sorry, OpenStack client in my env is little old.

**Review by samirjorina — APPROVED** [review]

**Pull request merged**